Gaijinizer patch/Map047.txt

```diff
@@ -696,10 +705,10 @@ ambush.#MTLed
 > CONTEXT: Map047/events/40/pages/3/0/Dialogue/hyakki
 > CONTEXT: Map261/events/40/pages/1/0/Dialogue/hyakki
 > CONTEXT: Map261/events/40/pages/3/0/Dialogue/hyakki
-Go around the end of this and if you go a little,
-it's a convenience store. \\C[2]The rest area may be a
-bit far from now, I recommend that you save and
-prepare for the time being.#MTLed
+Just ahead and around the corner, the store's a
```

Comment:
Hmm, I'd phrase it like this...
>_**If you go around the corner, the store's just a
bit further. \\C[2]The next rest area's
a bit far from here, so I recommend saving and
preparing there while you have the chance.**_